docs/global/modules/core/pages/howto/mps-model-server-plugin.adoc

```diff
@@ -0,0 +1,41 @@
+= How-To run the light `model-server` in MPS as plugin
+:navtitle: mps-model-server-plugin
+
+
+You can run a *light version* of the `model-server` directly in MPS via the dedicated *MPS as Modelix Model Server Plugin*, which is published over at the https://plugins.jetbrains.com/plugin/22834-mps-as-modelix-model-server[JetBrains Marketplace^].
+
+NOTE: More information on the component in general can be found in the xref:core:reference/component-mps-model-server-plugin.adoc[corresponding component reference].
+
+[IMPORTANT]
+====
+A light model-server does not provide the full features as an independent `model-server` instance will.
+More *advanced features* will not work with this light version (e.g. modelix specific repositories/branches, or the xref:howto/metrics.adoc[metrics]).
+
+If you want the full capabilities of the `model-server`, check out xref:howto/usage-model-api-gen-gradle.adoc[how to start it standalone].
+====
+
+
+== How to Install the Plugin
```

Comment:
```suggestion
== How to install the Plugin
```

Reply:
All other docs use title case. I will not break this schema in this single page.

Reply:
Then the H1 also needs to be title case. That's where I came from.